### ISSUE TITLE
Remove compression option

### ISSUE THREAD

**Issue by mrzone64** [angristan/openvpn-install#872]
The compress option in openvpn is a security risk and is [deprecated in openvpn 2.5.](https://community.openvpn.net/openvpn/wiki/DeprecatedOptions?__cf_chl_jschl_tk__=81d43c1379bc99410c149c9a216299b448772486-1626176871-0-AZ3rAwCTeevLCgkgzOm1AZN-pJ61mdxOmkkcZHFeIRAmz3J-iK_z_gwiAqcBYan34oMJnLsdxqLAWiuiE04xm0NXXJkETvISkOl-eN_nQ1SSgHYQ2kke2XP_Y_G_Eo9bZVrlaYpPL5O_M0tvUHkP2-0Dr6zWYjYIt_neauGYkDlRXU50IPAcWkwRA30Al4BiMVDu2p3uiCXD-VwPkw2B8fnHq5YHQ2jyCz7U7Iunq8pCi4YX3QIc7OqMP0HsXszW72bw222aRaWuEHuS3R1n13ilB814FK9KmxsVKdf29ubzLCci0MlU9xvNrDolcFXpR9MZ9QMoO4tWq_PfFpcBN0efDq8-nMx6eGgcXadwhxXI53r-CnAqxHTLR7POvamS_7mt0Lvabm7eIclesOLo9hlr7QPYxIetpmUu_yyoS98P1yl25H5YZ2ExMBpjtNH3JQ#Option:--compress)

Info about security risk [HERE](https://community.openvpn.net/openvpn/wiki/VORACLE#:~:text=The%20compression%20feature%20in%20OpenVPN,to%20as%20a%20compression%20frame%20.)

**Comment by angristan** [comment]
https://github.com/angristan/openvpn-install#compression :)

**Comment by mrzone64** [comment]
Yes, but should there not be a question about enabling compression when creating a new cert?
Compression:
1 Enable compresion (not supported in openvpn 2.5)
2 No compression

**Comment by angristan** [comment]
I guess we could remove compression entirely 🙂

**Comment by TinCanTech** [comment]
OpenVPN intend to remove compression, eventually.

https://community.openvpn.net/openvpn/wiki/Compression
